tests/CribblyBackend.Test.Support/Extensions/ControllerBaseExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using System.Net.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CribblyBackend.Test.Support.Extensions
+{
+    public static class ControllerBaseExtensions
+    {
+        public static void AddTestUser(this ControllerBase controller, string authId, string email = "")
+        {
+            var user = TestData.NewIdentityUser(authId, email);
+            controller.ControllerContext = new ControllerContext();
+            controller.ControllerContext.HttpContext = new DefaultHttpContext { User = user };
+        }
+
+        public static void AddHeader(this ControllerBase controller, string header, string value)
+        {
+            var ctx = controller.ControllerContext.HttpContext;
```

Comment:
Whoa there buddy, that looks like a name you'd find in go code. Have you been writing go code lately?